### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by shuyama1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 ## 4.25.0 (Unreleased)
 
+IMPROVEMENTS:
+* bigquery: added `connection_id` to `external_data_configuration` for `google_bigquery_table` ([#4365](https://github.com/hashicorp/terraform-provider-google-beta/pull/4365))
+* cloudfunctions2: Made `google_cloudfunctions2_function.event_trigger.service_account_email` and `google_cloudfunctions2_function.service_config.service_account_email` configurable. (beta) ([#4367](https://github.com/hashicorp/terraform-provider-google-beta/pull/4367))
+* composer: promoted `config.master_authorized_networks_config` in `google_composer_environment` resource to GA. ([#4355](https://github.com/hashicorp/terraform-provider-google-beta/pull/4355))
```
Comment (slevenick) [inline CHANGELOG.md:6]: Remove as it is related to GA only

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 ## 4.25.0 (Unreleased)
 
+IMPROVEMENTS:
+* bigquery: added `connection_id` to `external_data_configuration` for `google_bigquery_table` ([#4365](https://github.com/hashicorp/terraform-provider-google-beta/pull/4365))
+* cloudfunctions2: Made `google_cloudfunctions2_function.event_trigger.service_account_email` and `google_cloudfunctions2_function.service_config.service_account_email` configurable. (beta) ([#4367](https://github.com/hashicorp/terraform-provider-google-beta/pull/4367))
```
Comment (slevenick) [inline CHANGELOG.md:5]: Remove (beta), maybe reword this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+* composer: promoted `config.master_authorized_networks_config` in `google_composer_environment` resource to GA. ([#4355](https://github.com/hashicorp/terraform-provider-google-beta/pull/4355))
+* compute: added `advanced_options_config` to `google_compute_security_policy` ([#4354](https://github.com/hashicorp/terraform-provider-google-beta/pull/4354))
+* compute: added `cache_key_policy` field to `google_compute_backend_bucket` resource ([#4349](https://github.com/hashicorp/terraform-provider-google-beta/pull/4349))
+* compute: added `include_named_cookies` to `cdn_policy` on `compute_backend_service` resource ([#4358](https://github.com/hashicorp/terraform-provider-google-beta/pull/4358))
+* compute: added internal IPv6 support on `google_compute_network` and `google_compute_subnetwork` ([#4368](https://github.com/hashicorp/terraform-provider-google-beta/pull/4368))
+* container: add `managed_prometheus` to `monitoring_config` in `google_container_cluster` (beta) ([#4373](https://github.com/hashicorp/terraform-provider-google-beta/pull/4373))
```
Comment (slevenick) [inline CHANGELOG.md:11]: added

**Review by slevenick — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shuyama1) [reply CHANGELOG.md]: removed!

File: CHANGELOG.md (hunk 2 above)
Comment (shuyama1) [reply CHANGELOG.md]: Done! Thanks!

File: CHANGELOG.md (hunk 3 above)
Comment (shuyama1) [reply CHANGELOG.md]: Done.

**Review by slevenick — APPROVED** [review]
